lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -160,14 +174,7 @@ public function getDefaultLength(AbstractPlatform $platform)
      */
     public static function getType($name)
```

Comment:
Is this and other extracted methods supposed to be marked `@deprecated`?

Reply:
Eventually yes, but not at this moment - there is no way to use the registry directly, it's only used internally. We may design different approach.